### PR TITLE
Making the FIPS tests' output in report.txt more clear/readable

### PR DESCRIPTION
**Pull request by patrikcerbak**
I added `set +x` command to the `listCryptoAlgorithms()` and `listCryptoProviders()` methods, so the output in `report.txt` files contains less garbage. 

Before these changes, the file contained whole Java classes (which were saved into variables, so when the whole command was printed, the Java classes were there). Now, it mostly just contains the desired result.

### CODE REVIEW

**Comment by judovana** [comment]
Is it possible to echo the whole  command to stderr (or manually to global-stdouterr.txt) and then to execute it in +x/-x block?

**Comment by patrikcerbak** [comment]
> Is it possible to echo the whole command to stderr (or manually to global-stdouterr.txt) and then to execute it in +x/-x block?

What do you think? Is it ok now?

File: containersQa/testlib.bash [hunk 1]
```diff
@@ -990,12 +990,22 @@ function validateManualSettingFipsWithNoCrash() {
 
 function listCryptoAlgorithms() {
   skipIfJreExecution
-  runOnBaseDirBash "echo '$checkAlgorithmsCode' > /tmp/CheckAlgorithms.java && echo '$cipherListCode' > /tmp/CipherList.java && \
-                    javac -d /tmp /tmp/CheckAlgorithms.java /tmp/CipherList.java && java -cp /tmp CheckAlgorithms list algorithms"
+  set +x
+
+  commandAlgorithms="echo '$checkAlgorithmsCode' > /tmp/CheckAlgorithms.java && echo '$cipherListCode' > /tmp/CipherList.java && javac -d /tmp /tmp/CheckAlgorithms.java /tmp/CipherList.java && java -cp /tmp CheckAlgorithms list algorithms"
+  runOnBaseDirBash "$commandAlgorithms"
+  echo "runOnBaseDirBash $commandAlgorithms" >> $REPORT_DIR/global-stdouterr.log
```
Comment (judovana) [inline containersQa/testlib.bash:997]: please switch it. first echo it, then run :)

**Comment by patrikcerbak** [comment]
What about now?

**Comment by judovana** [comment]
> What about now?

Pls switch that echo and command itself. First echo, then cmd. Otherwise perfect:)

**Comment by judovana** [comment]
perfection!

**Review by judovana — APPROVED** [review]

**Pull request merged**